designs/tutorial.stanza

```diff
@@ -14,19 +14,31 @@ defpackage ocdb/designs/tutorial :
 
 val board-shape = RoundedRectangle(35.0, 30.0, 0.5)
 
+; this is a simple usage of the LM317A LDO where the component is instantiated with a simple
+; resistor divider feedback 
 pcb-module simple-lm317a-regulator (v-out:Double) :
+  port pwr-in : power 
```

Comment:
For the very first, 5 minute version of the tutorial I started without ports.

There will be a sequence of tutorials building up to the full test fixture design, and I was going to cover ports in the one immediately following. What do you think about that?

It is weird to have a module with no ports, but I was trying to minimize the overall number of concepts delivered in 5 minutes.

Reply:
Yea, this is a good question -- I think that ports are useful because if someone copies the simple regulator as a template for their design, it will not be useful without ports. My mindset was that the tutorial code will be copied and extended by the user because it is a convenient jumping off point. Hence the inclusion of ports.

Reply:
Let's focus this design on the quick start tutorial. 

I think that design templates are *super* important, but we do have a better way to create new projects with the new extension features.

There will be a version with ports in the same file no matter what. It does sound like we can avoid ports in the simple regulator.

Reply:
Ok, np.